src/runner/covid_runner.cpp

```diff
@@ -0,0 +1,75 @@
+#include "faabric/util/environment.h"
+#include <wasm/WasmModule.h>
+
+#include <boost/filesystem/operations.hpp>
+#include <boost/filesystem/path.hpp>
+#include <faabric/scheduler/Scheduler.h>
+#include <faabric/util/config.h>
+#include <faabric/util/func.h>
+#include <faaslet/FaasletPool.h>
+
+#include <module_cache/WasmModuleCache.h>
+#include <wamr/WAMRWasmModule.h>
+
+#include <faabric/util/files.h>
+
+int main(int argc, char* argv[])
```

Comment:
I'll delete the covid runner once we're done with developing this use-case, but it's very useful for now.